### PR TITLE
Tuesday November 10 2020 Meeting

### PR DESCRIPTION
**Pull request by nkohen**
Please comment on this PR or DM with things we should discuss or other changes that should be made to this document.

Please make sure to review and give feedback on the first two sections in particular on this one (for example, if you want to be added to the Individual Updates). There are some new things we will be trying as per our discussion after the previous meeting.

Once the meeting has happened I will add meeting notes and then once they have been reviewed this will be merged into meetings directory.

### CODE REVIEW

File: meeting/meeting_november_10_2020.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+**#Status_Update_Interrupt**
+
+* The initial [Oracle Proposal](https://github.com/discreetlogcontracts/dlcspecs/pull/55/) has been merged!
+  * Exact serialization and signing algorithm not fully specified by the current doc
+    * **#Discussion**
+    * https://github.com/discreetlogcontracts/dlcspecs/issues/111
+  * We need a doc for client-side oracle validation
+    * https://github.com/discreetlogcontracts/dlcspecs/issues/97
+    * Antoine has volunteered to do this
+* Any other oracle things worth discussion/mention?
```
Comment (benthecarman) [inline meeting/meeting_november_10_2020.md:52]: Tagged hashes?

File: meeting/meeting_november_10_2020.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  * https://github.com/discreetlogcontracts/dlcspecs/issues/109
+* **#Discussion**
+  * Anything about updates not discussed? Anything else?
+
+## P2P Network Considerations
+
+**#Query(Lloyd, all)**
+
+* Last meeting we discussed using TOR or BIP 342 for solving problems like port forwarding
+  * Lloyd was potentially interested in pursuing BIP 342
```
Comment (benthecarman) [inline meeting/meeting_november_10_2020.md]: 324*

File: meeting/meeting_november_10_2020.md (hunk 1 above)
Comment (nkohen) [reply meeting/meeting_november_10_2020.md:52]: This falls under the `Exact serialization and signing algorithm` discussion

File: meeting/meeting_november_10_2020.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+**#Status_Update_Interrupt** - In this mode I (or someone else) will be bringing everyone up to speed on a given topic, but in a setting where interruptions such as questions and discussions are expected and welcome.
+Discussions here should try to remain brief.
+
+**#Discussion** - This tag explicitly opens the floor up for anyone to ask questions or make comments.
+Note that discussion tags are always optional and often times no one will have anything to say, this tag merely marks places were there is the potential for fruitfull in-meeting discussions.
+
+**#Query([names])** - This tag explicitly opens the floor up for a specific person or group of people, generally to answer a question.
+
+**#Goal([statement])** - This tag marks the agenda's intention to have a discussion with a specific purpose to achieve. These discussions may take longer.
```
Comment (ariard) [inline meeting/meeting_november_10_2020.md]: Thanks for it ! Maybe extracting it as a README.md for dlcspecs/meeting/ :) ?

I think you may dissociate clearly _meeting chairman_, which has the job to make sure the discussion is structured, progressing towards resolution and not stalling in noise and _tag holder_, which is speaking on some subject ?

File: meeting/meeting_november_10_2020.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+    * A couple months ago Lloyd [mentioned](https://github.com/discreetlogcontracts/dlcspecs/issues/50#issuecomment-693101689) his anticipated approach, any updates?
+* Adding TLV Streams for extensibility
+  * Does this even make sense given the way we are using TLVs?
+  * https://github.com/discreetlogcontracts/dlcspecs/issues/73
+* Pretty Pictures Wanted!
+  * https://github.com/discreetlogcontracts/dlcspecs/issues/77
+  * **#Query(all)**
+    * Anybody interested or know someone who might be?
+    * Is there anything we can do to get this to happen? Maybe post to telegram or tweet?
+* SIGHASH discussion seems to have gone stale
```
Comment (ariard) [inline meeting/meeting_november_10_2020.md:116]: For this one, we should consider it for 0.2, for now I expect to be too much complexity.

File: meeting/meeting_november_10_2020.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+  * https://github.com/discreetlogcontracts/dlcspecs/issues/91
+  * I (Nadav) personally have been unable to follow this discussion, if there are any take-aways or proposals to be made, I will need them written and PR'd or put on that issue so that I can spend time understanding it
+  * **#Goal(Get a volunteer to write up take-away/proposal if applicable)**
+* "Key Material Compromised" Message
```
Comment (ariard) [inline meeting/meeting_november_10_2020.md:120]: I can write a sound proposal for this.

**Review by ariard — COMMENTED** [review]
Can we add a discussion on setting up a mailing list ?

I think we start to have a lot of discussions with deep context ("Key Material Compromised") and I expect mailing list to be a better tool than GH issues, also a wider communication channel for the booming DLC community.

File: meeting/meeting_november_10_2020.md (hunk 3 above)
Comment (nkohen) [reply meeting/meeting_november_10_2020.md]: I think my plan is to add these details and any other changes after the meeting depending on how things go and what people think but I like your idea of (at that time), separating this out into its own doc.

**Pull request merged**